src/main/java/org/sunbird/assessment/service/CohortsServiceImpl.java

```diff
@@ -337,6 +337,7 @@ private void processChildContentId(String givenContentId, List<String> assessmen
 					if (participantMap.containsKey(userId)) {
 						OpenSaberApiUserProfile userProfile = (OpenSaberApiUserProfile) participantMap.get(userId);
 						CohortUsers user = new CohortUsers();
+						user.setUserId(userProfile.getUserId());
```

Comment:
This is not the proper solution... the API getting called is needs to be replaced with SB User Search API.